config/config.go

```diff
@@ -2,49 +2,107 @@
 package config
 
 import (
-	"context"
+	"time"
 
-	"github.com/micro/go-micro/v3/config/loader"
-	"github.com/micro/go-micro/v3/config/reader"
-	"github.com/micro/go-micro/v3/config/source"
+	"github.com/micro/go-micro/v3/store"
 )
 
+type Values interface {
+	Get(path string, options ...Option) Value
+	Set(path string, val interface{}, options ...Option)
+	Delete(path string, options ...Option)
+}
+
 // Config is an interface abstraction for dynamic configuration
 type Config interface {
-	// provide the reader.Values interface
-	reader.Values
-	// Init the config
-	Init(opts ...Option) error
-	// Options in the config
-	Options() Options
-	// Stop the config loader/watcher
-	Close() error
-	// Load config sources
-	Load(source ...source.Source) error
-	// Force a source changeset sync
-	Sync() error
-	// Watch a value for changes
-	Watch(path ...string) (Watcher, error)
-}
-
-// Watcher is the config watcher
-type Watcher interface {
-	Next() (reader.Value, error)
-	Stop() error
+	Values
 }
 
-type Options struct {
-	Loader loader.Loader
-	Reader reader.Reader
-	Source []source.Source
+// NewConfig returns new config
```

Comment:
I don't think we need this in the top level, can you please move to go-micro/config/store

Reply:
I don't get this

Reply:
The implementation should't be in the top level package, only the interface. We'll import the implementations separately from within micro.